examples/plot_02_overview_skore_ui.py

```diff
@@ -104,7 +104,7 @@
 my_project_ui.get("my_string")
```

Comment:
Maybe we could outline that strings are interpreted as markdown by default and user can use markdown syntax out of the box ?

Reply:
It is not said after?

<img width="366" alt="Capture d’écran 2024-10-30 à 17 35 01" src="https://github.com/user-attachments/assets/b45e8e76-be4c-40a0-bd6b-a25e7eb345d5">

Reply:
Ok my bad.